app/src/main/java/com/epfl/beatlink/ui/search/SearchScreen.kt

```diff
@@ -0,0 +1,133 @@
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.epfl.beatlink.model.profile.ProfileData
+import com.epfl.beatlink.model.spotify.objects.SpotifyTrack
+import com.epfl.beatlink.model.spotify.objects.State
+import com.epfl.beatlink.ui.navigation.BottomNavigationMenu
+import com.epfl.beatlink.ui.navigation.LIST_TOP_LEVEL_DESTINATION
+import com.epfl.beatlink.ui.navigation.NavigationActions
+import com.epfl.beatlink.ui.navigation.Screen
+import com.epfl.beatlink.ui.search.components.FullSearchBar
+import com.epfl.beatlink.ui.search.components.GradientTitle
+import com.epfl.beatlink.ui.search.components.PartyCard
+import com.epfl.beatlink.ui.search.components.ProfileCard
+import com.epfl.beatlink.ui.search.components.SongCard
+import com.epfl.beatlink.ui.search.components.StandardLazyRow
+
+@Composable
+fun SearchScreen(navigationActions: NavigationActions) {
+  val song1 =
+      SpotifyTrack(
+          name = "Song1",
+          trackId = "1",
+          cover = "cover1",
+          duration = 120,
+          popularity = 80,
+          state = State.PAUSE)
+  val listOfTrendingSongs = listOf(song1)
+  val sortedByPopularitySongs = listOfTrendingSongs.sortedByDescending { it.popularity }
+
+  val song2 =
+      SpotifyTrack(
+          name = "Song2",
+          trackId = "1",
+          cover = "cover1",
+          duration = 120,
+          popularity = 80,
+          state = State.PAUSE)
+  val listOfTrendingSongs2 = listOf(song2)
+  val sortedByPopularitySongs2 = listOfTrendingSongs2.sortedByDescending { it.popularity }
+
+  val profile1 =
+      ProfileData(
+          username = "@username1", name = "Name1", bio = "bio1", links = 3, profilePicture = 1)
+  val listOfProfiles = listOf(profile1)
+
+  Scaffold(
+      topBar = { FullSearchBar(navigationActions) },
+      bottomBar = {
+        // Bottom navigation bar
+        BottomNavigationMenu(
+            onTabSelect = { route -> navigationActions.navigateTo(route) },
+            tabList = LIST_TOP_LEVEL_DESTINATION,
+            selectedItem = navigationActions.currentRoute())
+      },
+      modifier = Modifier.testTag("searchScreen")) { paddingValues ->
+        Column(
+            modifier =
+                Modifier.testTag("searchScreenColumn")
+                    .fillMaxSize()
+                    .padding(paddingValues)
+                    .verticalScroll(rememberScrollState())
+                    .background(color = Color.White)) {
```

Comment:
here you should use the app backgroup (eg. lightThemeBackground) instead of plain white